backend/lcfs/web/api/other_uses/views.py

```diff
@@ -123,7 +123,6 @@ async def get_other_uses_paginated(
 
 @router.post(
     "/save",
-    response_model=Union[OtherUsesSchema, DeleteOtherUsesResponseSchema],
```

Comment:
Union types have some funky behaviour in Pydantic, so in this case it was applying the new validator to the DeleteOtherUsesResponseSchema even though it wasnt defined on it.